src/components/SystemManagedList.js

```diff
@@ -187,6 +187,7 @@ export default class SystemManagedList extends Component {
       for (let filterType in activeFilters) {
         const value = activeFilters[filterType]
         const fieldValue = this.getValue(definition, filterType)
+        if (!value) return true
         if (value === 'presence') {
           if (!fieldValue) return false
         } else if (value === 'absence') {
```

Comment:
can we get 'presence' and 'absence' options still?

btw We use this in the 'Share as Notice file' workflow

Reply:
Do you mean to hide that Filter from the list?
I think it could be useful, maybe we should ask for some user feedback on this

Reply:
I mean with the spdx picker in place we lose the ability to filter by presence of and absence of a license - I think we should retain them, especially with the workflow for share as notice

Reply:
Actually the Spdx Filter we set the same value of the old Dropdown list in the `activeFilters` object.
That should allow the filter `presence` and `absence` still to work.
Did you found a different behavior?

Reply:
I've checked the deploy and i've understand what you mean.
What do you suggest to bring those options back?

Reply:
@dabutvin we need still to solve the mentioned problem here. Do you have any suggestion?

Reply:
Sorry for the delay.

Let's augment the spdx picker with additional values, the way we do with NOASSERTION/NONE.
It would probably be valuable to separate them by a line.

```
NONE
NOASSERTION
PRESENCE OF
ABSENCE OF
--------------
OBSD
AAL
...
```